webrisk_client.go

```diff
@@ -447,7 +447,7 @@ func (sb *WebriskClient) LookupURLsContext(ctx context.Context, urls []string) (
 
 	for _, req := range reqs {
 		// Actually query the Web Risk API for exact full hash matches.
-		resp, err := sb.api.HashLookup(ctx, req)
+		resp, err := sb.api.HashLookup(ctx, req.HashPrefix, req.ThreatTypes)
```

Comment:
can you rename all variables named sb to wr?

Reply:
Done.